examples/libs/stack-check/Makefile

```diff
@@ -0,0 +1,9 @@
+CONTIKI=../../..
+CONTIKI_PROJECT = main
+CFLAGS += -DPROJECT_CONF_H=\"project-conf.h\"
+
+CONTIKI_WITH_IPV6 = 1
```

Comment:
I think we need `MAKE_NET_IPV6` here

Reply:
ok, I'll check - was copy paste from Contiki...